jest.config.js

```diff
@@ -37,4 +37,5 @@ module.exports = {
     "jest-watch-typeahead/testname",
   ],
   reporters: ["default"].concat(ENABLE_TEST_RESULTS ? "jest-junit" : []),
+  verbose: true,
```

Comment:
Could we remove this `verbose: true`? Now Jest prints too much when I run it locally.

Reply:
How about set to `isCI`, so `yarn test` will print less, and `yarn test --verbose` can still print more

Reply:
#7866